### PR TITLE
HighFive integration tests

### PR DESCRIPTION
**Pull request by ohm314**
Setup integration tests for highfive with MorphIO and libsonata.
The tests should run on a nightly schedule and with a webhook trigger when a PR in highfive is merged.

### CODE REVIEW

**Comment by ferdonline** [comment]
Thanks @ohm314, I like this clean way of doing integration tests!
I would suggest using a `ci` folder with generic build/test scripts so that (1) we don't do bash programming inside a .yml file (2) we can reuse stuff more easily

**Review by ferdonline — APPROVED** [review]

**Review by matz-e — APPROVED** [review]
Neat!

File: .github/workflows/integration.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+name: HighFive_Integration_tests
+
+on:
+  push:
+    branches:
+      - ci_test
```
Comment (alexsavulescu) [inline .github/workflows/integration.yml:6]:
```suggestion
  push:
    branches:
      - master
```

**Review by alexsavulescu — COMMENTED** [review]
LGTM

File: .github/workflows/integration.yml (hunk 1 above)
Comment (alexsavulescu) [reply .github/workflows/integration.yml:6]: discussed offline, misunderstanding on my part

**Review by alexsavulescu — APPROVED** [review]

**Pull request merged**